### ISSUE TITLE
some questions about rambo.py

### ISSUE THREAD

**Issue by cclvr2** [yihaosun1124/OfflineRL-Kit#2]
Thanks guys  for providing such an elegant and well performed coda base. I have one question about Line 196-199 of rambo.py. 

        all_loss = self._adv_weight * adv_loss + sl_loss
        self._dynmics_adv_optim.zero_grad()
        all_loss.backward()
        self._dynmics_adv_optim.step()

I guess adv_loss is the Model Gradient (as defined in the paper) from its computations rule, but  it's then added as an adversary loss to the overall loss.  So why is it? 

I'm looking forward to your replies. Wish you guys all the best.
